index.bs

```diff
@@ -465,7 +480,10 @@ The following steps are taken:
 1. Let _entries_ be _registry_.[[RegistryData]].
 1. Let _pair_ be the entry in _entries_ such that _pair_.[[key]] is equal to _key_.
 1. If _pair_ exists, then throw a new *TypeError*.
-1. Let _entry_ be a new registry entry record { [[Key]]: _key_, [[State]]: "ready", [[Metadata]]: *undefined*, [[Fetch]]: *undefined*, [[Translate]]: *undefined*, [[Instantiate]]: *undefined*, [[Dependencies]]: *undefined*, [[Module]]: _module_ }.
+1. Let _result_ be a new promise resolved with _module_.
```

Comment:
directly creating the new record in "ready" stage, not need to add all other stages.